.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: CRC IRC Community Chat (suitable for short, simple casual discussions)
+    url: https://freenode.net/
```

Comment:
Is this the right URL?

Reply:
I just used the same URL as provided in https://github.com/crc-org/crc?tab=readme-ov-file#community

I can change it to any other URL decided by the team.

Reply:
No I mean the channel name is `#codeready` so there should be a clear direction to use that.

Reply:
Didn't we move this to liberachat?
However nobody from the team is watching this channel as far as I know, so I'm not sure it is useful to mention it.

Reply:
@cfergeau : Shall I remove this list element?

Reply:
Just saw the `codeready` reference but I didn't notice it. Maybe we can increase the font or change its placement? Just my opinion

Reply:
+1 with Rohan. I think we can remove this field

Reply:
We can always re-add after wer made a decision in regard to #4506